Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -356,6 +378,70 @@ private void CreateTriggerChangeLevel(PxVobTriggerChangeLevelData vob)
             }
         }
 
+        private void CreateTriggerList(PxVobTriggerListData vob)
+        {
+            var gameObject = new GameObject(vob.vobName);
```

Comment:
All your new methods share the exact same code base.

Please create just one method called (e.g.)
```CreateTrigger(PxVobTriggerData vobTrigger)```

Hint: Your classes are already derived from PxVobTriggerData